README.md

```diff
@@ -160,7 +160,7 @@ No Modules.
 | monitoring\_interval | The interval (seconds) between points when Enhanced Monitoring metrics are collected | `number` | `0` | no |
 | monitoring\_role\_arn | IAM role for RDS to send enhanced monitoring metrics to CloudWatch | `string` | `""` | no |
 | name | Name given resources | `string` | `""` | no |
-| password | Master DB password | `string` | `""` | no |
+| password | Master DB password (Note: When specifying a value here, the 'create_random_password' variable needs to be set to false)  | `string` | `""` | no |
```

Comment:
Please update `description` value in `variables.tf`, so that it matches README.md.

We use `terraform-docs` and `pre-commit` hooks to automatically update a part of the content of README.md from Terraform configs.